kolibri/plugins/management/assets/src/device_management/views/manage-permissions-page/user-full-name-cell.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+
+  <span>
+
+    <span class="icon" ref="icon">
+      <mat-svg
+        v-if="permissionType==='SUPERUSER'"
+        category="toggle"
+        name="star"
+        class="superuser"
+      />
+      <mat-svg
+        v-if="permissionType==='SOME_PERMISSIONS'"
+        category="toggle"
+        name="star"
+        class="some-permissions"
+      />
+      <mat-svg
+        v-if="permissionType==='NO_PERMISSIONS'"
+        category="social"
+        name="person"
+        class="no-permissions"
+      />
+      <ui-tooltip trigger="icon">
+        {{ tooltipText }}
+      </ui-tooltip>
+    </span>
+
+    <span class="align">{{ user.full_name }}</span>
+
+  </span>
+
+</template>
+
+
+<script>
+
+  import uiTooltip from 'keen-ui/src/UiTooltip';
+  import every from 'lodash/every';
+  import omit from 'lodash/omit';
+
+  function hasNoPermissions(ps) {
+    if (!ps) return true;
+    return every(omit(ps, ['user']), p => !p);
```

Comment:
would appreciate more descriptive variable names than `ps` and `p`, or a comment indicating what form they take, to make the code easier to scan